package.json

```diff
@@ -61,7 +61,7 @@
   },
   "build": {
     "appId": "im.riot.app",
-    "electronVersion": "8.0.1",
+    "electronVersion": "8.0.3",
```

Comment:
Hmm, this upgrade is only safe once we've done something about `will-navigate` and SSO. We could add the `will-navigate` temporary removal in this PR, or hold this one until the SSO PR merges, or ...?

Reply:
Blocking on SSO PR for simplicity

Reply:
Okay. Do note you'll conflict with yourself, since the SSO PR upgrades to 8.0.2 here...

Reply:
somehow this hasn't conflicted O_O